website/docs/d/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,146 @@
+
+Use this data source to access information about an existing Exadata Infrastructure.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_exadata_infrastructure" "example" {
+  name                = "existing"
+  resource_group_name = "existing"
+}
+
+output "id" {
+  value = data.azurerm_oracle_exadata_infrastructure.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Exadata Infrastructure.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Exadata Infrastructure exists.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Exadata Infrastructure.
+
+* `activated_storage_count` - The requested number of additional storage servers activated for the Exadata infrastructure.
+
+* `additional_storage_count` - The requested number of additional storage servers for the Exadata infrastructure.
+
+* `available_storage_size_in_gbs` - The available storage can be allocated to the cloud Exadata infrastructure resource, in gigabytes (GB).
+
+* `compute_count` - The number of compute servers for the cloud Exadata infrastructure.
+
+* `cpu_count` - The total number of CPU cores allocated.
+
+* `customer_contacts` - A `customer_contacts` block as defined below.
+
+* `data_storage_size_in_tbs` - Size, in terabytes, of the DATA disk group.
+
+* `db_node_storage_size_in_gbs` - The local node storage allocated in GBs.
+
+* `db_server_version` - The software version of the database servers (dom0) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `display_name` - The user-friendly name for the cloud Exadata infrastructure resource. The name does not need to be unique.
+
+* `estimated_patching_time` - A `estimated_patching_time` block as defined below.
+
+* `last_maintenance_run_id` - The [OCID](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm) of the last maintenance run.
+
+* `lifecycle_details` - Additional information about the current lifecycle state.
+
+* `lifecycle_state` - CloudExadataInfrastructure lifecycle state.
+
+* `location` - The Azure Region where the Exadata Infrastructure exists.
+
+* `maintenance_window` - A `maintenance_window` block as defined below.
+
+* `max_cpu_count` -  The total number of CPU cores available.
+
+* `max_data_storage_in_tbs` - The total available DATA disk group size.
+
+* `max_db_node_storage_size_in_gbs` - The total local node storage available in GBs.
+
+* `max_memory_in_gbs` - The total memory available in GBs.
+
+* `memory_size_in_gbs` - The memory allocated in GBs.
+
+* `monthly_db_server_version` - The monthly software version of the database servers (dom0) in the cloud Exadata infrastructure. Example: `20.1.15`
+
+* `monthly_storage_server_version` - The monthly software version of the storage servers (cells) in the cloud Exadata infrastructure. Example: `20.1.15`
```

Comment:
```suggestion
* `monthly_db_server_version` - The monthly software version of the database servers (dom0) in the Cloud Exadata Infrastructure. Example: `20.1.15`

* `monthly_storage_server_version` - The monthly software version of the storage servers (cells) in the Cloud Exadata Infrastructure. Example: `20.1.15`
```

Reply:
Changed to Cloud Exadata Infrastructure.